packages/core-assortments/src/module/configureAssortmentsModule.ts

```diff
@@ -255,8 +255,14 @@ export const configureAssortmentsModule = async ({
       return Assortments.findOne(selector, {});
     },
 
-    findAssortments: async ({ limit, offset, ...query }) => {
-      const assortments = Assortments.find(buildFindSelector(query), {
+    findAssortments: async ({ limit, offset, queryString, ...query }) => {
+      const filters = { ...query };
```

Comment:
seems inconsistent, buildFindSelector should be extended to take a queryString not findAssortments.